core/finality-grandpa/src/environment.rs

```diff
@@ -637,7 +637,7 @@ where
 		round: u64,
 		state: RoundState<Block::Hash, NumberFor<Block>>,
 		base: (Block::Hash, NumberFor<Block>),
-		votes: Vec<SignedMessage<Block>>,
+		votes: &HistoricalVotes<Block::Hash, NumberFor<Block>, Self::Signature, Self::Id>,
```

Comment:
Why do we need this changes?

Reply:
It is an implementation of the `Environment` trait from `finality-grandpa`. The trait definition is changed, so the implementation needs to be updated.

Reply:
Ahh you updated grandpa, makes sense!